Zuora AQuA User Guide (#509)
mulesoft/docs-connectors: modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-design-center.adoc

= Zuora AQuA Design Center Configuration - Mule 4
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:imagesdir: ../../assets/images/

Anypoint Design Center's Flow Designer enables you to create apps visually. For more information, see the xref:design-center::fd-tour.adoc[Flow Designer Tour].

To create an app in Flow Designer:

* Configure the input source (trigger) for your app.
* Add the connector as a component to process the input for the app.

== Configure the Input Source Trigger

. In Design Center, click *Create*.
. Click *Create new application*.

> . Specify a value for *Project name*, and click *Create*.

Reviewer: ```suggestion
. Specify a value for *Project name* and click *Create*.
```

. Exit from *Let's get started* by clicking *Go straight to canvas*.
. Click on the trigger card.
. Search for and select *HTTP Listener* or *Scheduler*.

> . Complete the fields with the values you want to configure for your trigger. For more information on field values, see the xref:zuora-aqua/zuora-aqua-connector-reference.adoc[Zuora AQuA Connector Reference - Mule 4].

Reviewer: ```suggestion
. Complete the fields with the values you want to configure for your trigger. 
For more information about field values, see the xref:zuora-aqua/zuora-aqua-connector-reference.adoc[Zuora AQuA Connector Reference - Mule 4].
```

== Configure the Zuora AQuA Target Component

> To add a Zuora AQuA connector as a component:

Reviewer: ```suggestion
To add Zuora AQuA Connector as a component:
```

. Click *+* next to the trigger card.
. In *Select a component*, search for the connector name.
. Select the connector.
. Select an operation for the connector.
. If needed, specify values that are required in the *General* tab.
+
For example, if you choose *Add Attachments* as the operation, you must enter a value for the *Associated Object Type* and *Associated Object Key fields*.
+
. Click *Click here to set it up* to set the required fields.

> . Specify access information to the connector resource. For this connector, you can use only basic authentication, as described below.

Reviewer: ```suggestion
. Specify access information to the connector resource. 
For this connector, you can use only basic authentication, as described in Parameters for the Basic Auth Configuration.
```

. Click *Test* to test the connection.

=== Parameters for the Basic Auth Configuration

Use these parameters to configure basic authentication:

[%header%autowidth.spread]
|===
|Field |Description
|*Username*| Zuora username
|*Password*| Zuora password
|*Base URI* a| Base path of the AQuA API for your instance. For example: + 
`https://apisandbox-api.zuora.com:443/`
|===

image::zuora-aqua/aqua-config.png[Zuora API Configuration]

== Next Step

> After creating an app in Flow Designer, see the xref:zuora-aqua/zuora-aqua-connector-examples.adoc[Zuora AQuA Examples] section to experiment with the Zuora AQuA connector.

Reviewer: ```suggestion
After creating an app in Flow Designer, see the xref:zuora-aqua/zuora-aqua-connector-examples.adoc[Zuora Examples] section to experiment with Zuora AQuA Connector.
```
